mocha.el

```diff
@@ -54,21 +53,23 @@ If MOCHA-FILE is specified run just that file otherwise run MOCHA-PROJECT-TEST-D
           mocha-options " "
           (or mocha-file mocha-project-test-directory)))
 
-;;;###autoload
 (defun mocha-run (&optional mocha-file)
   "Run mocha in a compilation buffer.
 
 If MOCHA-FILE is specified run just that file otherwise run MOCHA-PROJECT-TEST-DIRECTORY"
   (compile (mocha-command mocha-file))
-  (kill-buffer "*mocha tests*")
+  (when (get-buffer "*mocha tests*")
+    (kill-buffer "*mocha tests*"))
   (pop-to-buffer "*compilation*")
   (rename-buffer "*mocha tests*"))
 
+;;;###autoload
```

Comment:
So is the convention to just put autoloads on any public interactive functions? I didn't really understand where to put them from reading the manual, and most packages I looked at just put them on the mode definition.